src/test/java/controllers/ApiControllerDocTesterTest.java~origin_master

```diff
@@ -1,46 +0,0 @@
-/**
- * Copyright (C) 2013 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package controllers;
-
-
-import org.junit.Test;
-
-import ninja.NinjaDocTester;
-import org.doctester.testbrowser.Request;
-import org.doctester.testbrowser.Response;
-import org.hamcrest.CoreMatchers;
-import static org.hamcrest.CoreMatchers.containsString;
-import static org.junit.Assert.assertThat;
-
-public class ApiControllerDocTesterTest extends NinjaDocTester {
-    
-    String URL_INDEX = "/";
-    String URL_ACES_UP = "/AcesUp";
-    
-    @Test
```

Comment:
The test is removed. Is it because the test is doubled? or we decide not to use the method